lib/tags.js

```diff
@@ -6,6 +6,57 @@ const Base = require('./base')
  * Tags - Used for communication with intercomClient's tags objects
  * @class
  */
-class Tags extends Base {}
+class Tags extends Base {
+
+  /**
+   * Create or update a tag on a company/user object, passes data through to intercom
+   * @param {Object} tag
+   * @param {Object} tag.id - Tag unique identifier
+   * @param {Object} tag.name - Tag name
+   * @returns {Promise} - Returns success if intercom is disabled, otherwise
+   *   returns results of create command in intercom
+   * @see https://developers.intercom.io/reference#create-and-update-tags
+   */
+  create (tag) {
+    return this.wrap('create', tag)
+  }
+
+  /**
+   * Create or update a tag on a company/user object, passes data through to intercom
+   * @param {Object} tag
+   * @param {Object} tag.id - Tag unique identifier [required]
+   * @param {Object} tag.name - Tag name
```

Comment:
Webstorm picks these up when you use the method.  You can see the parameters (and their types) by pressing ⌘+P after typing the method and a (